e2e/playwright/README.md

```diff
@@ -29,17 +29,17 @@ npm run test
 
 #### 1. Run docker container using latest focal version
 
-In this directory at `mattermost-webapp/e2e/playwright`, run docker container
+Change to root directory at `mattermost-webapp`, run docker container
 
 ```
-docker run -it --rm -v "$(pwd):/e2e-playwright/" --ipc=host mcr.microsoft.com/playwright:v1.27.0-focal /bin/bash
+docker run -it --rm -v "$(pwd):/mattermost-webapp/" --ipc=host mcr.microsoft.com/playwright:v1.27.0-focal /bin/bash
```

Comment:
Changed to root directory so it will pick up dependencies from `/packages` directory such as client and types.